### ISSUE TITLE
Agrypnia Excitata

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10858]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 30890351 The clinical hallmark of FFI, Agrypnia Excitata is a definite cliniconeurophysiological condition characterized by: 1) slow wave sleep loss with disruption of the physiological sleep-wake cycle, 2) a day and night, motor, sympathetic and aminergic overactivity and 3) peculiar episodes of oneiric stupor.
(....) Outside FFI, Agrypnia Excitata has been canonically observed inother diseases presenting different pathophysiological mechanisms, such as Morvan Syndrome (MS) and Delirium Tremens (DT)

PMID: 35501502 Based on the recent literature and collective empirical knowledge of the experts, agrypnia excitata, neuroophthalmologic dysfunction, weight loss, and hyperthermia were suggested as potential items to incorporate into the revised diagnostic criteria for FFI. The term agrypnia excitata (agrypnia meaning “chasing sleep away” referring to sleep loss of organic origin, and excitata referring to the motor and autonomic activation) defnes a generalized overactivation syndrome characterized by severe and persistent insomnia and marked motor and autonomic sympathetic activation (including weight loss and hyperthermia)

**Parent term (use hpo.jax.org/app)**
Sleep abnormality HP:0002360

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Morvan syndrome ORPHA:83467
Fatal familial insomnia ORPHA:466

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f I wonder if this is more of a disease than a specific phenotypic feature? https://pubmed.ncbi.nlm.nih.gov/23423537/

**Comment by pnrobinson** [comment]
@carolina-f 
I think that Agrypnia excitata is not a good HPO term because it bundles several different phenotypic features together (and the authors say "is a syndrome "). This would thus be a better ORPHA term!
